tests/apps/gens/test_gens_api.py

```diff
@@ -0,0 +1,63 @@
+"""
+    Tests for GensAPI
+"""
+
+from pathlib import Path
+from typing import Dict
+
+from cg.apps.gens import GensAPI
+from cg.utils.commands import Process
+
+
+def test_instantiate(gens_config: dict):
+    """Test to instantiate GensAPI."""
+    # GIVEN a gens api config dictionary
+
+    # WHEN instantiating a gens api
+    gens_api = GensAPI(gens_config)
```

Comment:
Use type hints in tests as well. It makes it easier to understand the test and what is expected